packages/fuel-indexer-tests/tests/e2e/indexing_postgres.rs

```diff
@@ -364,11 +364,8 @@ async fn test_can_index_metadata_when_indexer_macro_is_called_postgres() {
             .await
             .unwrap();
 
-    assert_eq!(
```

Comment:
`IndexMetadataEntity` is now created via `::new()` so the ID is no longer deterministic (since it includes `time`)